gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/query/DataFetcherNode.java

```diff
@@ -2,6 +2,10 @@
 
 import org.hypertrace.gateway.service.entity.query.visitor.Visitor;
 import org.hypertrace.gateway.service.v1.common.Filter;
+import org.hypertrace.gateway.service.v1.common.OrderByExpression;
+
+import java.util.Collections;
+import java.util.List;
 
 /**
  * Node in the execution tree that applies the encapsulated filters and fetches attributes
```

Comment:
Edit the doc to mention it also applies order/limit/offset conditionally if required